src/generators/typescript/TypeScriptRenderer.ts

```diff
@@ -156,17 +156,24 @@ ${renderedLines}
 
   renderProperty(propertyName: string, property: CommonModel, type: PropertyType = PropertyType.property): string {
     const formattedPropertyName = this.nameProperty(propertyName, property);
+
+    const desc = property.getFromOriginalInput('description').trim();
+    const examples = property.getFromOriginalInput('examples');
+    const formattedExamples = `@example ${examples?.join ? examples.join(', ') : examples}`
+    const doc = this.renderComments(`${desc || ''}\n${examples ? formattedExamples : ''}`.trim());
+    const formattedDoc = desc || examples ? `${doc}\n` : '';
```

Comment:
Remember, any of these properties fetched from the original input might not exist at all, so to use them you need extensive checks ensuring that you find anything 🙂